pages/studio/[[...index]].tsx

```diff
@@ -1,22 +1,120 @@
+import { useEffect, useState } from "react";
 import Head from "next/head";
+import Image from "next/image";
 import { NextStudio } from "next-sanity/studio";
-import { NextStudioHead } from "next-sanity/studio/head";
+import { useRouter } from "next/router";
 import { StudioLayout, StudioProvider } from "sanity";
 import config from "sanity.config";
+import { Text } from "@sanity/ui";
+import { NEXT_PUBLIC_APP_URL } from "studio/config";
+import AutologinPrepage from "studio/components/AutologinPrepage";
 
 export default function StudioPage() {
+  const maxRetries = 10;
+  const router = useRouter();
+
+  const [isReady, setIsReady] = useState(true);
+  const [isAutologin, setIsAutologin] = useState(false);
+  const [retryAutologin, setRetryAutologin] = useState(0);
+
+  useEffect(() => {
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Since you're using the `router` here, let's use the `query` which is I believe is the same values from the result of here.